### ISSUE TITLE
Vague link labels need to be made more specific

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#2744]
As per Trusted Tester for link purpose, The purpose of each link can be determined from any combination of the link text, accessible name, accessible description, and/or programmatically determined link context.

- [Revenues](https://revenuedata.doi.gov/how-revenue-works/revenues): multiple links are labeled as Review the Details but go to different sites. These labels need to be more clear on what they go to.
- [AML Reclamation](https://revenuedata.doi.gov/how-revenue-works/aml-reclamation-program/): Many of the in-text links go to the same resource FAQs page but are labeled very differently. These should be consistently labeled or we remove some of the links and only link to thatexternal  site once on the page.
- [Native American Ownership](https://revenuedata.doi.gov/how-revenue-works/native-american-ownership-governance/): some in-text links are not descriptive of what they link to

**Comment by cthomasONRR** [comment]
@eelzi-ONRR I understand the requested fixes to Revenues, but would like to discuss the issues you list for AML Reclamation and Native American Ownership.

For AML Reclamation, I see the multiple links to the same site. I think I could either just have it linked once and remove the other link instances. Is that ok?

For Native American Ownership, all the links seem very descriptive to me, so I'm not sure what you want here. None are generic "review the details" or anything like that.

**Comment by cthomasONRR** [comment]
Updates in PR #2850. Closing issue.
